From 3cda3df87c6c2362a92003e93b128d4388a0def6 Mon Sep 17 00:00:00 2001
From: "Loc Phuoc Pham.Lam" <loc.lam.px@renesas.com>
Date: Fri, 23 Aug 2019 13:38:59 +0700
Subject: [PATCH 289/345] arm64: dts: renesas: r8a774b1-hihope-rzg2n-ex: Enable
 PCIE0 and PCIE1

This patch enables PCIE0 and PCIE1 for the HiHope board.

Signed-off-by: Loc Phuoc Pham.Lam <loc.lam.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774b1-hihope-rzg2n-ex.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774b1-hihope-rzg2n-ex.dts b/arch/arm64/boot/dts/renesas/r8a774b1-hihope-rzg2n-ex.dts
index d2fb32c..ccab6bc 100644
--- a/arch/arm64/boot/dts/renesas/r8a774b1-hihope-rzg2n-ex.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774b1-hihope-rzg2n-ex.dts
@@ -15,3 +15,11 @@
 	compatible =	"hoperun,hihope-rzg2-ex", "hoperun,hihope-rzg2n",
 			"renesas,r8a774b1";
 };
+
+&pciec0 {
+	status = "okay";
+};
+
+&pciec1 {
+	status = "okay";
+};
-- 
2.7.4

